src/Processor/Braintree/Query/SubscriptionQuery.php

```diff
@@ -60,6 +60,13 @@ public function whereSubscriptionIsPending(): self
         return $this;
     }
 
+    public function whereSubscriptionIdIs(string $id): self
```

Comment:
Can we just call this `whereId`?

Reply:
Or should we just call this `getById` and have it return the `Subscription`? Would we ever expect multiple subscriptions to come out of this, and would we care about additional criteria?

Reply:
@brianium changed pushed - I went with `whereId` since this is in the query and it keeps consistency with the other functions in there.